rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -219,14 +219,80 @@ Delete global variables that will not be used anymore. Variables linked in confi
 . In the *Global Variables* view of the *Process Automation* module, click the *Delete* icon image:delete-icon.png[trash symbol,1.5%,1.5%] in the table row of the global variable you want to delete.
 . Confirm the deletion.
 
-=== Create Credentials
+=== Manage Credentials
 
 Credentials store login data for applications or web services. They can be used with different processes. Passwords are encrypted.
 
+The following types of credentials can be created:
+
+* *AWS Credentials*
++
+for accessing Amazon Web Services
+* *User Account Credentials*
++
+for accessing all other kinds of applications
+
+You can link credentials anywhere that a globe symbol is displayed. A link symbol indicates an existing link. You can change and delete links.
+
+Editing or deleting a credential affects all configurations which use the credential.
+
+Check the usage of a credential before changing or deleting it.
+
+==== Create Credentials
+
+Create credentials with which a bot can login to an external application during process runs.
+
 . In the *Credential Pool* view of the *Process Automation* module, click *Create*.
-. Complete the form *Create a New Credential*.
+. Complete the form *Create a New Credential*:
+* *Phase affiliation*:
++
+Phases in which the credential is permitted to be deployed. When users create or change the credential, each user's individual privileges determine which phases the user can select:
++
+** *Test*:
++
+The credential can be deployed in the test phase.
++
+** *Production*:
++
+The credential can be deployed in the production phase.
+
+* *Type*:
++
+Type of the global credential. The type of credential determines where it can be used. The original value defined in RPA Builder is typed. Select one of the following types:
++
+** *AWS Credentials*:
++
+The credential is used for accessing Amazon Web Services. Fill in client ID, client secret and region. Using a session token is optional.
+** *User Account Credentials*
++
+The credential is used for accessing all other kinds of applications. Fill in the username and password
 . Click *OK*.
 
+==== Check the Usage of a Credential
+
+Before editing a credential, check its usage to avoid inadvertent side-effects.
+
+. In the *Credential Pool* view of the *Process Automation* module, click the *Usage* icon image:usage-icon.png[binoculars symbol,1.5%,1.5%] in the table row of the credential you want to check.
+
+A window with a table shows you in which configurations the credential is used.
```

Comment:
Suggestion:
A window with a table shows you the configurations in which the credential is used.